### PR TITLE
Facetoface chalhoubuniversityfix

### PR DESCRIPTION
**Pull request by marcghaly**
Hi @mattporritt,
This PR has point one of issue #56, point 2 I would need some information as where we would get DB values
It also contains commits from issue 56 - rebased over latest master. (3 commits)
cheers !

### CODE REVIEW

**Comment by mattporritt** [comment]
Hi @marcghaly ,
Thanks for looking at this. For point 2 that you've mentioned. I'd go back to the reported and ask them for some more info on what they're after.
Can you also have a look at the failing automated checks? For the style/doc check I'd just fix the errors not the warnings.

Cheers,
Matt P

**Comment by marcghaly** [comment]
Hi @mattporritt, I have fixed errors - and asked in the other PR,

Cheers !

File: lang/en/facetoface.php [hunk 1]
```diff
@@ -413,6 +413,7 @@
 
 You will receive a reminder [reminderperiod] business days before this course.
 ';
+$string['setting:defaultconfirmationmessagedefault2'] = '<p><b>Thank you for signing up!</b>';
```
Comment (danmarsden) [inline lang/en/facetoface.php:416]: this looks a bit wrong... something missing? :-)

File: lib.php [hunk 2]
```diff
@@ -2135,7 +2137,7 @@ function facetoface_user_cancel($session, $userid=false, $forcecancel=false, &$e
  * @returns string Error message (or empty string if successful)
  */
 function facetoface_send_notice($postsubject, $posttext, $posttextmgrheading,
-                                $notificationtype, $facetoface, $session, $userid) {
+                                $notificationtype, $facetoface, $session, $userid,$htmlmessage) {
```
Comment (danmarsden) [inline lib.php:2140]: looks like you're missing a space after the comma - I'm sure this code doesn't comply well, but we shouldn't be adding new issues ;-)

File: lib.php [hunk 3]
```diff
@@ -2186,7 +2188,8 @@ function facetoface_send_notice($postsubject, $posttext, $posttextmgrheading,
                                                           $user, $session, $session->id);
                 $body = facetoface_email_substitutions($posttext, format_string($facetoface->name), $facetoface->reminderperiod,
                                                        $user, $session, $session->id);
-                $htmlbody = ''; // TODO.
+                 $htmlmessage = facetoface_email_substitutions($posttext, $facetoface->name, $facetoface->reminderperiod, $user, $session, $session->id);
```
Comment (danmarsden) [inline lib.php:2191]: tabs look wrong here - extra space in front of $htmlmessage ?

**Comment by danmarsden** [comment]
generally looks ok to me, but few comments above - watch your spacing - there were a few other lines in there that were not looking aligned....

**Pull request closed without merging**